WordPress/src/main/res/values-ko/strings.xml

```diff
@@ -31,86 +31,86 @@ Language: ko_KR
-    <string name="gutenberg_native_double_tap_to_edit_this_value">“두 번 탭하여 이 값 편집”</string>
-    <string name="gutenberg_native_double_tap_to_change_the_value_using_slider">“두 번 탭하여 슬라이더를 사용하여 값 변경”</string>
-    <string name="gutenberg_native_double_tap_to_add_a_block">“두 번 탭하여 블록 추가”</string>
-    <string name="gutenberg_native_double_tap_and_hold_to_edit">“두 번 탭하고 길게 눌러 편집”</string>
-    <string name="gutenberg_native_current_value_is_s">“현재 값은 %s입니다.”</string>
-    <string name="gutenberg_native_content">“콘텐츠…”</string>
-    <string name="gutenberg_native_clear_all_settings">“모든 설정 지우기”</string>
-    <string name="gutenberg_native_choose_from_device">“기기에서 선택”</string>
-    <string name="gutenberg_native_annotations_sidebar">“주석 사이드바”</string>
-    <string name="gutenberg_native_an_unknown_error_occurred_please_try_again">“알 수 없는 오류가 발생했습니다. 다시 시도하세요”</string>
-    <string name="gutenberg_native_alt_text">“대체 텍스트”</string>
-    <string name="gutenberg_native_add_video">“비디오 추가”</string>
-    <string name="gutenberg_native_add_url">“URL 추가”</string>
-    <string name="gutenberg_native_add_link_text">“링크 텍스트 추가”</string>
-    <string name="gutenberg_native_add_image_or_video">“이미지 또는 비디오 추가”</string>
-    <string name="gutenberg_native_add_image">“이미지 추가”</string>
-    <string name="gutenberg_native_add_block_here">“블록을 여기에 추가”</string>
-    <string name="gutenberg_native_add_annotation">“주석 추가”</string>
-    <string name="gutenberg_native_add_a_description">“설명 추가”</string>
+    <string name="gutenberg_native_wordpress_media_library">워드프레스 미디어 라이브러리</string>
+    <string name="gutenberg_native_we_are_working_hard_to_add_more_blocks_with_each_release_in_the_m">당사는 각 릴리스에 더 많은 블록을 추가하기 위해 노력하고 있습니다. 그 사이에 웹에서 이 글을 편집할 수도 있습니다.</string>
+    <string name="gutenberg_native_unsupported">지원되지 않음</string>
+    <string name="gutenberg_native_ungroup">그룹 해제</string>
+    <string name="gutenberg_native_translate">번역</string>
+    <string name="gutenberg_native_title">제목:</string>
+    <string name="gutenberg_native_tap_to_hide_the_keyboard">탭하여 키보드 숨김</string>
+    <string name="gutenberg_native_tap_here_to_show_help">탭하여 도움말 보기</string>
+    <string name="gutenberg_native_take_a_video">비디오 촬영</string>
+    <string name="gutenberg_native_take_a_photo_or_video">사진 찍기 또는 비디오 촬영</string>
+    <string name="gutenberg_native_take_a_photo">사진 촬영</string>
+    <string name="gutenberg_native_start_writing">글쓰기 시작…</string>
+    <string name="gutenberg_native_size">크기:</string>
+    <string name="gutenberg_native_sidebar_title_plugin">사이드바 제목 플러그인</string>
+    <string name="gutenberg_native_show_section">섹션 표시</string>
+    <string name="gutenberg_native_s_isn_t_yet_supported_on_wordpress_for_ios">\'%s\'은(는) iOS용 워드프레스에서 아직 지원되지 않습니다.</string>
+    <string name="gutenberg_native_s_isn_t_yet_supported_on_wordpress_for_android">\'%s’은(는) Android용 워드프레스에서 아직 지원되지 않습니다.</string>
```

Comment:
This one looks weird to me, even though it doesn't seem to be related to the issue we're fixing.

Reply:
@Tug this one still looks weird

Reply:
It's fine I think, this is how the translator wrote it and even though opening quote don't match the closign one it's not a huge deal imo